hypervisor/boot/acpi_base.c

```diff
@@ -264,11 +257,10 @@ uint16_t parse_madt(uint32_t lapic_id_array[MAX_PCPU_NUM])
 uint8_t parse_madt_ioapic(struct ioapic_info *ioapic_id_array)
 {
 	uint8_t ret = 0U;
-	struct acpi_table_rsdp *rsdp = NULL;
+	const struct acpi_table_rsdp *rsdp = get_rsdp();
```

Comment:
Typically, we only parse ACPI tables when CONFIG_ACPI_PARSE_ENABLED is defined.  Here it seems to be unconditionally.  Does it work when CONFIG_ACPI_PARSE_ENABLED is not defined?